ShipMonitor/ShipLoadoutEvent.cs

```diff
@@ -31,15 +31,15 @@ static ShipLoadoutEvent()
         public string ship { get; private set; }
         public string shipname { get; private set; }
         public string shipident { get; private set; }
-        public long value => hullvalue + modulesvalue;
-        public long hullvalue { get; private set; }
-        public long modulesvalue { get; private set; }
+        public long? value => hullvalue + modulesvalue ?? null;
```

Comment:
We don't need the `?? null` because of a C# feature called "lifted operators". 
https://stackoverflow.com/questions/3370110/what-are-lifted-operators